packages/lit-dev-content/site/css/docs-api.css

```diff
@@ -40,7 +40,7 @@
 }
 copy-button {
   height: 48px;
-  opacity: 0.5;
+  opacity: 0.75;
```

Comment:
We have two selectors for `copy-button`?

Reply:
Yeah, should probably refactor. Though there's an advantage to at least having the `height` set in the page stylesheets, because that way the sizing is correct even before the element upgrades...